### PR TITLE
@rpath/libqdlt.dylib fix for MacOS

### PR DESCRIPTION
**Pull request by bunty95**
Fixed @rpath/libqdlt.dylib would not be copied into "DLTViewer.app" for MacOS.

No longer requires that the libqdlt to be placed into /usr/local on MacOS

### CODE REVIEW

File: .github/workflows/BuildPR.yml [hunk 1]
```diff
@@ -41,61 +41,3 @@ jobs:
         with:
           name: DLTViewer-${{ matrix.macos }}-x86_64
           path: build/dist/DLTViewer*.tgz
-
-  buildLinux:
```
Comment (hannesa2) [inline .github/workflows/BuildPR.yml]: Please enable this again

File: .github/workflows/Release.yml [hunk 2]
```diff
@@ -37,90 +37,25 @@ jobs:
           name: DLT-Mac
           path: DLT-macOS.zip
 
-  buildLinux:
-    name: Build ${{ matrix.ubuntu }}
```
Comment (hannesa2) [inline .github/workflows/Release.yml]: Please enable this again

File: .github/workflows/Release.yml [hunk 3]
```diff
@@ -37,90 +37,25 @@ jobs:
...
-        uses: actions/upload-artifact@v4
-        with:
-          name: DLT-Windows-${{ matrix.task }}
-          path: DLT-*.zip
+
 
   createRelease:
     name: Create release
     runs-on: ubuntu-20.04
-    needs: [ buildMac, buildLinux, buildWindows ]
```
Comment (hannesa2) [inline .github/workflows/Release.yml]: I guess, you did this just for development purpose, right ?

**Review by hannesa2 — CHANGES_REQUESTED** [review]

File: .github/workflows/Release.yml (hunk 3 above)
Comment (bunty95) [reply .github/workflows/Release.yml]: Yes was done to make development faster. re enabled windows and ubuntu

**Comment by bunty95** [comment]
All workflows in BuildPR and Release have been reenabled

**Review by hannesa2 — APPROVED** [review]

**Comment by hannesa2** [comment]
I approved it, but I can't verify because I've only an arm64 Mac and there it doesn't build.

@bunty95 I try to build on arm64 https://github.com/COVESA/dlt-viewer/pull/485 any help is welcome

**Comment by bunty95** [comment]
@hannesa2 let me investigate #485, will respond in that issue

**Pull request merged**